apps/wfo-ui-surf/components/WfoAvatar/styles.ts

```diff
@@ -0,0 +1,38 @@
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+import { css } from '@emotion/react';
+
+export const getStyles = (theme: EuiThemeComputed) => {
```

Comment:
In addition to the discussion below i would like to suggest to replace all the magic numbers/colors with a value from the theme. Or at least as much as possible